plugins/must-gather/skills/must-gather-analyzer/scripts/run-camgi.sh

```
@@ -0,0 +1,241 @@
+# Function to check if command exists
+command_exists() {
+    command -v "$1" >/dev/null 2>&1
+}
+
+# Function to stop camgi containers
+stop_camgi() {
+    echo -e "${BLUE}Stopping camgi containers...${NC}"
+
+    # Check for podman
+    if command_exists podman; then
+        local containers=$(podman ps -q --filter ancestor=quay.io/elmiko/okd-camgi)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)